tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
+    var toggler = event.target;
+    if (toggler.dataset.toggle !== 'collapse') {
+      return;
+    }
+    var toggleTarget = document.getElementById(toggler.dataset.target);
+
+    if (toggleTarget.classList.contains('collapse')) {
+      toggleTarget.classList.remove('collapse');
```

Comment:
I don't think there's a good answer to this, as there's nothing to stop the presence of the `collapse` class and the value of the `aria-expanded` attribute from getting out of sync. I'm happy to test whichever one you prefer, but it is easier to test for the presence of the classname IMO.